tensorflow/cc/framework/cc_op_gen.cc

```diff
@@ -57,6 +57,16 @@ string GetPath(const string& dot_h_fname) {
   return result;
 }
 
+// Converts: some/path/to/file.xx
+// to: file
+// (note that suffix is removed)
+string GetFilename(const string& path) {
```

Comment:
That's right. This shouldn't affect the binary distribution, assuming it is stripped properly.